qiskit/circuit/quantumcircuit.py

```diff
@@ -687,6 +687,46 @@ def compose(self, other, qubits=None, clbits=None, front=False, inplace=False):
 
         return dest
 
+    def tensor(self, other):
+        """Tensor ``self`` with ``other``.
+
+        Args:
+            other (QuantumCircuit): The other circuit to tensor this circuit with.
+
+        Returns:
+            QuantumCircuit: the tensored circuit (returns None if inplace==True).
+        """
```

Comment:
Can you add a note/example to the docstring on the endianness convention? What you have in the release note would be good.

Reply:
Added in 340f673 👍